corehq/apps/users/views/mobile/users.py

```diff
@@ -737,7 +737,11 @@ def create_mobile_worker(self, in_data):
 
         is_valid = lambda: self.new_mobile_worker_form.is_valid() and self.custom_data.is_valid()
         if not is_valid():
-            return {'error': _("Forms did not validate")}
```

Comment:
Wow, that old error was really not helpful.

Reply:
Yeah unfortunately it still doesn't make it to the front end - I guess maybe because the front end expects all validation to happen in JS? 

But at least you can see it in the network tab now and we're setup to add it in the future...